app/services/gpo_reminder_sender.rb

```diff
@@ -1,16 +1,11 @@
 class GpoReminderSender
+  LOCAL_DATABASE_TIMEOUT = 60_000
```

Comment:
this seems like the kind of thing better suited to be a config value than a constant, IMO, because the size of the tables, etc changes over time and having to do a code deploy to change that is not optimal